e2e-tests/commands/insurance/account/create-an-account-and-application-and-sign-in.js

```diff
@@ -8,14 +8,15 @@
  * 6) Complete and submit the "enter security code" form
  * 7) Check we are on the "all sections" application page.
  * @param {String} Account email address
+ * @param {String} Company number/companies house number to use for application creation
  */
-const createAnAccountAndApplicationAndSignIn = (emailAddress) =>
+const createAnAccountAndApplicationAndSignIn = (emailAddress, companyNumber) =>
```

Comment:
do we need a default company number here?

Reply:
No, this commnand calls `createAnApplication` which has a default company number, which is overridden if a company number is provided